### PR TITLE
chore(docs): Release notes 2022.13

### LINKED ISSUE #8177: release notes 2022.13
rel_nts_22_13

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #8177

### CODE REVIEW

**Comment by evwilkin** [comment]
@tlabaj it looks like one of the integration tests failed for this PR
```
<testcase name="Tab Demo Test Adjusts tabs showing on resize" time="0.0000" classname="Adjusts tabs showing on resize">
    <failure
        message="Timed out retrying after 4000ms: Expected to find content: &apos;Tab item 6&apos; but never did." 
        type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: Expected to find content: 'Tab item 6' but never did.
    at eval (http://localhost:3000/__cypress/tests?p=cypress/integration/tabshorizontaloverflow.spec.ts:142:37)
at Array.forEach (<anonymous>)
    at Context.eval (http://localhost:3000/__cypress/tests?p=cypress/integration/tabshorizontaloverflow.spec.ts:141:96)]]>
    </failure>
</testcase>
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -3,6 +3,135 @@ id: Release notes
...
+  - Added enhancements and updated types in wizard next version ([#7915](https://github.com/patternfly/patternfly-react/pull/7915))
+  - Added component unit tests for wizard next version ([#7731](https://github.com/patternfly/patternfly-react/pull/7731))
+  - Added explicit props due to react version upgrade in next version wizard footer wrapper ([#8171](https://github.com/patternfly/patternfly-react/pull/8171))
+
+### Charts
+- Fixed interactive legend highlighting not working as expected ([#8035](https://github.com/patternfly/patternfly-react/pull/8035))
+- Updated Victory dependencies ([#7986](https://github.com/patternfly/patternfly-react/pull/7986))
+
+### Log viewer
+Added prop `initialIndexWidth` to support custom line number column size ([#7939](https://github.com/patternfly/patternfly-react/pull/7939))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:101]:
```suggestion
- Added prop `initialIndexWidth` to support custom line number column size ([#7939](https://github.com/patternfly/patternfly-react/pull/7939))
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -3,6 +3,135 @@ id: Release notes
...
+  - Added sub component descriptions ([#7974](https://github.com/patternfly/patternfly-react/pull/7974))
+- **Code editor:**
+  - Added sub component descriptions ([#7974](https://github.com/patternfly/patternfly-react/pull/7974))
+- **Date picker:**
+  - Changed popover props type to partial ([#8002](https://github.com/patternfly/patternfly-react/pull/8002))
+  - Added sub component descriptions ([#7974](https://github.com/patternfly/patternfly-react/pull/7974))
+  - Fixed invalid input with multiple validators ([#7992](https://github.com/patternfly/patternfly-react/pull/7992))
+  - Fixed clearing validation error from outside ([#8036](https://github.com/patternfly/patternfly-react/pull/8036))
+- **Dropdown:**
+  - Added content to dropdown examples. ([#7784](https://github.com/patternfly/patternfly-react/pull/7784))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:41]:
```suggestion
  - Added content to dropdown examples ([#7784](https://github.com/patternfly/patternfly-react/pull/7784))
```

**Review by evwilkin — CHANGES_REQUESTED** [review]
Missing bullet point under Log viewer section, and added a note to delete a period for consistency.

**Review by evwilkin — APPROVED** [review]
LGTM

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**